linux_os/guide/services/ssh/file_permissions_sshd_private_key/ansible/shared.yml

```diff
@@ -7,8 +7,16 @@
 {{% set find_command_base = 'find -H /etc/ssh/ -maxdepth 1 -user root -regex ".*_key$" -type f' %}}
 {{% set dedicated_ssh_groupname = groups.get("dedicated_ssh_keyowner", {}).get("name") %}}
 
+{{% if product in ["sle12", "sle15"] %}}
+{{% set find_command = find_command_base + ' -group root -perm /u+xs,g+xws,o+xwrt' %}}
+{{% set permissions_mode = 'u-xs,g-xws,o-xwrt' %}}
+{{% else %}}
+{{% set find_command = find_command_base + ' -group root -perm /u+xs,g+xwrs,o+xwrt' %}}
+{{% set permissions_mode = 'u-xs,g-xwrs,o-xwrt' %}}
+{{% endif %}}
+
 - name: Find root:root-owned keys
-  command: '{{{ find_command_base }}} -group root -perm /u+xs,g+xwrs,o+xwrt'
+  command: '{{{ find_command }}}'
```

Comment:
Please, use the FQCN for Ansible modules in all tasks: https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible

I know it was not part of the PR, but it would be great to take this opportunity to update the Ansible Playbook.

Reply:
Thanks, @marcusburghardt ,

I've made the suggested updates and also a small refactoring of my changes.

Best regards